packages/schematics/angular/utility/project.ts

```diff
@@ -35,6 +43,23 @@ export function getProject<TProjectType extends ProjectType = ProjectType.Applic
   return workspace.projects[projectName] as WorkspaceProject<TProjectType>;
 }
 
+export function getProjectConfig(host: Tree, path: string) {
+  const configName = path.split('/').pop();
+  const configFile = host.read(path);
+  if (!configFile) {
+    throw new Error(`Couldn't find project config (${configName}).`);
```

Comment:
This seem to be related to tsconfig. I would make it clear what config it is that it cannot be found.

Reply:
`configName` displays the file name. Am I missing something?